src/modules/seal/components/SealWidgetPane.tsx

```
@@ -40,6 +41,14 @@ export function SealWidgetPane(sharedProps: SharedProps) {
     setSelectedSealUrnIndex(urn?.urnIndex !== undefined ? Number(urn.urnIndex) : undefined);
   };
 
+  // Reset detail pane urn index when widget is mounted
+  useEffect(() => {
+    const urnIndexParam = searchParams.get(QueryParams.SealUrnIndex);
+    setSelectedSealUrnIndex(
+      urnIndexParam ? (isNaN(Number(urnIndexParam)) ? undefined : Number(urnIndexParam)) : undefined
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)